sdx_lc/handlers/sdx_controller_msg_handler.py

```diff
@@ -94,8 +95,12 @@ def process_sdx_controller_json_msg(self, msg):
                     self.logger.info(
                         "Check your configuration and make sure OXP service is running."
                     )
-                self.logger.info(f"Status from OXP: {oxp_response}")
-                self.send_conn_response_to_sdx_controller(service_id, oxp_response)
+                self.logger.info(
+                    f"Status from OXP: {oxp_response} - {oxp_response.text}"
+                )
+                self.send_conn_response_to_sdx_controller(
+                    service_id, msg_json["operation"], oxp_response
```

Comment:
Is it possible that msg_json doesn't contain "operation" field? For safety, maybe we should use msg_json.get("operation") instead of directly accessing "operation".

Reply:
Hi @congwang09 good point! thanks for point this out. Indeed, this won`t happen because of the check on line 86:
```
            if msg_json.get("operation") == "post":
```

The `msg_json["operation"]` has to be defined at that point.

Reply:
I see 👍